easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-1.2.0.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = "CellRanger-ATAC"
+version = "1.2.0"
+
+homepage = "https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac"
+description = """Cell Ranger ATAC is a set of analysis pipelines that process 
+Chromium Single Cell ATAC data."""
+
+toolchain = SYSTEM
+
+# Download manually from https://cf.10xgenomics.com/releases/cell-atac/cellranger-atac-1.2.0.tar.gz
```

Comment:
Visiting this URL results in an XML error about missing cookies, navigating via `homepage` ends up on a mandatory registration page.

Reply:
I have changed the commented out link to point to homepage of the software. you should be able to use the wget or curl command provided on that page to download it directly on a system.

Reply:
Can't test this as I don't want to register for a software that has license restrictions. I guess it's as good as it gets.